README.md

```
@@ -59,6 +59,88 @@ The following API key parameters are supported:
+steps:
+  - name: Run promptfoo evaluation
+    uses: promptfoo/promptfoo-action@v1
+    with:
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+      config: 'promptfooconfig.yaml'
+      # All environment variables above are automatically available
+```
+
+#### API Keys
+
+**Option 1: Action Inputs**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)